twindle-cli/twitter/utils/tweet-utils.js

```diff
@@ -19,12 +19,14 @@ const {
  * @returns {string}
  */
 const extractTweetId = (tweet_url) =>
-  [...matchAll(tweet_url, /https?:\/\/twitter.com\/[a-zA-Z_]{1,20}\/status\/([0-9]*)/g)][0][1];
+  //[...matchAll(tweet_url, /https?:\/\/twitter.com\/[a-zA-Z_]{1,20}\/status\/([0-9]*)/g)][0][1];
+  tweet_url.split('/')[5]
 
 const extractScreenName = (tweet_url) =>
-  tweet_url
-    .substring(0, tweet_url.lastIndexOf("/status"))
-    .substring(tweet_url.lastIndexOf("/") + 1);
+  //tweet_url
+    //.substring(0, tweet_url.lastIndexOf("/status"))
+    //.substring(tweet_url.lastIndexOf("/") + 1);
+    tweet_url.split('/')[3]
```

Comment:
This part seems good. We need a better solution to find the userID, and this might be it.

Off course, handle the missing http case.

Or, in my opinion just `str.replace('https://', '').replace('http://', '')`, and adapt for that.

If you can convert above into a single regex, all the better, but its all fine